src/editor/Model/State.re

```diff
@@ -19,10 +19,12 @@ module Tab = {
 };
 
 type t = {
+  neovimActive: bool,
```

Comment:
This implementation seems to add a lot of complexity to the state, with these changes (along with the fact we have to maintain an independent set of tab state!). 

Is there a more minimal approach we could take? For example, could we have a special hidden buffer in Neovim represent the 'Home' / Welcome experience?

Reply:
not sure what you mean re independent tab state? do you mean the type? that seemed like something inherent to what we would want i.e. the ability to represent a GUI tab type for certain buffers vs an editor tab type?